homeassistant/components/eps/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""EPS integration."""
+import asyncio
+
+from pyepsalarm import EPS
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_TOKEN, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import EPSDataUpdateCoordinator
+
+PLATFORMS = [Platform.ALARM_CONTROL_PANEL]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up EPS config."""
+    token: str = entry.data[CONF_TOKEN]
+    username: str = entry.data[CONF_USERNAME]
+    password: str = entry.data[CONF_PASSWORD]
+    eps_api = EPS(token, username, password)
+
+    # Use the EPS siteId as a unique_id
+    try:
+        async with asyncio.timeout(10):
+            site = await hass.async_add_executor_job(eps_api.get_site)
+    except (asyncio.TimeoutError, ConnectionError) as ex:
+        raise ConfigEntryNotReady from ex
```

Comment:
This is done when creating the config entry, and used as unique id for the config entry.
Is there a reason we can't use `entry.unique_id` here?

Reply:
Hi @emontnemery , 
I am not sure what you mean exactly... Sorry I am a complete HA beginner :/
Thanks for the help !
Cheers